### PR TITLE
Unit tests for `EtlPipeline` Class methods

### LINKED ISSUE #268: Write unit tests for `EtlPiplines.py`
# Purpose

Write pytests for `EtlPipline.py` class and for every method that is part of it.

## Considerations

A dummy class will have to be initialized, use the mock class that is in the `etl_pipelines/tests/utils/mock_EtlPipline_obj.py`
This will have all the methods that the `EtlPipeline` class has. No need to test the abstract methods (the ones with `return None`) Since they will be tested in the other sub-classes.

## Completion Criteria

Write tests for:
- [ ] Initialization
- [ ] `load_data`
- [ ] `get_downloaded_data`
- [ ] `get_transformed_data`
- [ ] `validate_downloaded_data`

### PR DESCRIPTION
**Pull request by Kashike-Umemura**
# Description

Added tests to the `EtlPipeline` class methods, this includes the initialization as well.

Also added some new logging restrictions for the database import from s3.

## Types of changes

Testing (Addition of new tests)

## Checklist

**NOTE** The tests that were added here passes. The tests that were written in the past do NOT pass at this moment. There are other issues to address that.

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

## Further comments

Closes #268

### CODE REVIEW

File: database_initialization/queries/post_import_queries.py [hunk 1]
```diff
@@ -2709,6 +2709,14 @@
...
 ALTER DATABASE bcwat_dev RESET pg_audit.log_statement;
 ALTER USER "bcwat-api-admin" RESET pg_audit.log_statement;
+ALTER DATABASE bcwat_dev RESET pg_audit.log;
+ALTER USER "bcwat-api-admin" RESET pg_audit.log;
+ALTER DATABASE bcwat_dev RESET pg_audit.log_level;
+ALTER USER "bcwat-api-admin" RESET pg_audit.log_level;
+ALTER DATABASE bcwat_dev RESET log_min_duration_statement;
+ALTER USER "bcwat-api-admin" RESET client_min_messages;
+ALTER DATABASE bcwat_dev RESET log_min_duration_statement;
+ALTER USER "bcwat-api-admin" RESET client_min_messages;
```
Comment (LiamF-FS) [inline database_initialization/queries/post_import_queries.py:2719]: very nice!
